rocAL/rocAL/include/meta_data/meta_data_reader.h

```diff
@@ -41,7 +41,8 @@ enum class MetaDataReaderType
     CAFFE2_DETECTION_META_DATA_READER,
     TF_DETECTION_META_DATA_READER,
     VIDEO_LABEL_READER,
-    MXNET_META_DATA_READER
+    MXNET_META_DATA_READER,
+    FILE_LIST_META_DATA_READER
```

Comment:
Is the name crct? The other meta readers has metadata to read like bboxes, etc. But this reader is more like the folder based label reader so shouldn't it be called label reader instead of meta data reader?

Reply:
Its not folder based. Its a file list that should contain name of the file with space seperacted labels